x/ecocredit/server/server.go

```diff
@@ -83,6 +94,15 @@ func newServer(storeKey sdk.StoreKey, paramSpace paramtypes.Subspace,
 	if err != nil {
 		panic(err.Error())
 	}
+
+	s.batchesByProjectIDIndex, err = orm.NewIndex(batchInfoTableBuilder, BatchesByProjectIndex, func(value interface{}) ([]interface{}, error) {
+		projectID := value.(*ecocredit.BatchInfo).ProjectId
```

Comment:
just now noticing that all the indexerFunc's are implemented like this -- should we be checking for errors here? do we ever expect the `value` not to be of the type were casting to?

Reply:
Yeah, It makes sense to check for errors.

Reply:
ok cool, created issue here https://github.com/regen-network/regen-ledger/issues/664